### ISSUE TITLE
Functions (functions.php) not loaded when using wp_ajax from frontend

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#394]
```
When retrieving content from the frontend/theme using AJAX, the templates can't 
access get_custom_field(), because functions.php is not loaded.

WordPress recommends to make AJAX calls (even from front) using admin-ajax.php. 
But with this environment, is_admin() == TRUE, skipping the load of the 
accessors.

1. Prepare theme/functions.php with an AJAX handler
add_action('wp_ajax_nopriv_XXX', 'XXX');
add_action('wp_ajax_XXX', 'XXX');
function XXX()
{
    get_template_part('content');
    exit;
}

2. Make an AJAX call is the JS
$.ajax({
    url: '/wp-admin/admin-ajax.php',
    data: { action: 'XXX' },
    type: 'POST',
    complete: function(data) { … }
});

3. In the template, make a call to get_custom_field(…)
```

Original issue reported on code.google.com by `berenger...@gmail.com` on 5 Jul 2012 at 3:44

**Comment by GoogleCodeExporter** [comment]
```
Good point.  I hand't considered Ajax access.  If you're doing your own custom 
stuff via Ajax, for now you'll have to do a "require_once" of the CCTM's 
functions.php file.  
```

Original comment by `ever...@fireproofsocks.com` on 5 Jul 2012 at 11:44
- Changed state: **InReview**

**Comment by GoogleCodeExporter** [comment]
```
Ok, I've added this so it's always included.  This will be live in 0.9.6.1

If you need to tweak this now, you can edit the code in the loader.php
```

Original comment by `ever...@fireproofsocks.com` on 26 Jul 2012 at 2:24
- Changed state: **Fixed**
- Added labels: **Milestone-Release0.9.7**
